src/services/EmployeeService.java

```diff
@@ -0,0 +1,22 @@
+package services;
+
+import mappers.EmployeeMapper;
+import models.EmployeeModel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class EmployeeService {
+
+    public HashMap<String, EmployeeModel> get(String logInCode) {
+
+        CsvReader csvReader = new CsvReader();
+        final EmployeeMapper employeeMapper = new EmployeeMapper();
+        String employeeListPath = "/Users/joeyarbrough/Advanced-Java-Labs/Homework/Final-Project/src/csvFiles/employees.csv";
+
+        ArrayList<String> fetchedData = csvReader.fetch(employeeListPath);
+
+        return employeeMapper.map(fetchedData);
```

Comment:
❓ HMMmmm..... I wrote a big comment on your factory saying how it is likely not needed and that you should do something with it, but it looks like you are not even using the factory right now.  This is what i wanted to see as what you needed to do for option 1 of the 2 options that i gave you.   This being said, I would maybe just delete that unused factory and keep the code as is.